precompile/modules/initia_stdlib/sources/vip/vip.move

```diff
@@ -1202,6 +1234,9 @@ module initia_std::vip {
         );
 
         module_store.stage = stage + 1;
+
+        add_tvl_snapshot_internal(module_store);
```

Comment:
good to add comment

Reply:
good call! we have to lift readability up